### ISSUE TITLE
refactor: zipkin exporter: simplify code

### ISSUE THREAD

**Issue by pjanotti** [open-telemetry/opentelemetry-collector#216]
The zipkin exporter uses a reporter and other structures the makes the code more complex than it needs to be while having batching and other features that are not desired. Refactoring it to leverage `exporterhelper` will give the code better observability and will be consistent with other exporters. Some outline: 
```go
	ze, err := exporterhelper.NewTraceExporter(
		"zipkin",
		ze.PushTraceData,
		exporterhelper.WithSpanName("otelsvc.exporter.Zipkin.ConsumeTraceData"),
		exporterhelper.WithRecordMetrics(true))
```

**Comment by huikang** [comment]
@pjanotti , if this one has not been taken, I'd like to give it a shot? Thanks.

**Comment by pjanotti** [comment]
Hi @huikang nobody took this issue. It will be an welcome contribution.

I would recommend trying to separate the translation code from the code that sends the data to Zipkin, that should be a HTTP client and since we have the queued_retry processor you don't need to implement retry/buffer logic on it: basically an HTTP client that receives the data and then sends to the endpoint.

**Comment by pjanotti** [comment]
I assigned it to you @huikang - let's know if you have any questions (reach us out here or on gitter).

**Comment by huikang** [comment]
Sure, will do.

**Comment by pjanotti** [comment]
Fixed via #412
